api/src/test/java/marquez/db/SearchDaoTest.java

```diff
@@ -48,7 +59,51 @@ public static void setUpOnce(final Jdbi jdbi) {
     DbTestUtils.newDataset(jdbi, "time_ordering_1");
     DbTestUtils.newDataset(jdbi, "time_ordering_2");
 
-    DbTestUtils.newJobs(jdbi, NUM_OF_JOBS);
+    ImmutableSet<JobRow> jobRows = DbTestUtils.newJobs(jdbi, NUM_OF_JOBS);
+
+    // add a symlinked job - validate that the number of results is the same
```

Comment:
The comment mentions "validation", but this code is mostly setting up the test suite. Are we asserting that there's no exception thrown?

Reply:
I'll update the comment to clarify - the existing tests are unchanged because the symlinked job points to the new job and that's the one that is returned from the search results. Let me figure out how to make that clear and explicit in the test itself.

Reply:
See [this](https://github.com/MarquezProject/marquez/commit/48130fe0e6fee7d64c200e2243c18de809c69b20) commit for the update

Reply:
Nice! yeah, that’s a bit more clear, thanks!